bool is_shape_detected
float32 center_x
float32 center_y